scripts/helpers.php

```
@@ -0,0 +1,292 @@
+    if (!$splitArguments['dryRun']) {
+        return $splitArguments['options'];
+    }
+
+    return array_values(array_unique(array_merge($dryRunOptions, $splitArguments['options'])));
+}
+
+function resolve_lint_paths(array $pathArguments): array
+{
+    if ($pathArguments === []) {
+        return array_map(
+            static fn (string $name): string => Path::join('packages', $name),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Symfony\Component\Filesystem\Path' in method 'resolve_lint_paths'. (undefined)

(StaticAccess)